hepdata/ext/elasticsearch/config/record_mapping.py

```diff
@@ -107,7 +107,7 @@
             "institution": {
                 "type": "text"
             },
-            "defense_data": {
+            "defense_date": {
```

Comment:
Yes - we will need to reindex otherwise the new `defence_date` field won't be populated.

Reply:
Just reindex, or also recreate, i.e. `-rc True` option of the `hepdata utils reindex` CLI command?  I don't like to recreate unless necessary because records become unavailable until reindexed (although it should only take a few minutes).  With `-rc False` (the default), the records are still available during reindexing.

Reply:
I *think* it should be sufficient to reindex without recreating but I'd need to check what elasticsearch will do with the old `defense_date` field. I'll do that now.

Reply:
I was wrong earlier: the `defense_date` field should still be populated currently, even though it isn't in the mapping. We could check by going to http://ELASTICSEARCH_HOST:9200/hepdata-main/_mapping - if it is in there then we may not need to reindex at all.

I couldn't check with the `defense_data`/`defense_date` field as I don't have any records where it's populated, but from tests on other fields I think what will currently be the case on prod is that both fields will exist in the index: the former because it was created by the mapping, the latter because it was added when records were added. If we reindex without recreating the mapping, the `defense_data` field will still exist in the mapping, but will be empty. 

It would be good to reindex and recreate the index to tidy things up, but it's probably not mandatory.